config/routes.rb

```diff
@@ -1,7 +1,10 @@
 Rails.application.routes.draw do
+  devise_for :users, controllers: { registrations: 'registrations' }
   get '/restaurant', controller: :restaurant, action: :index
 
   resources :menu, only: [:index, :create, :new]
   resources :dishes, only: [:new, :show, :create]
 
+  root to: 'restaurant#index'
```

Comment:
Fixed, I saved line 3 instead of line 8.
Line 8 wasn't actually working/doing anything.